### ISSUE TITLE
Update High Contrast setup for Checkbox and Radio button

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-flutter#155]
based on updates from the accessibility team, the high contrast mode should be updated as follows:
* if disabled: nothing specific to do
* if enabled: important semantic components/elements (like checkbox or radio buttons) must use the "color.content.default" color token to be displayed in light & dark mode

Note that this "color.content.default" color can also be the color used for the hover state.
Note also that it is recommended to be able to distinguish the enabled and the hover states.
Therefore when the high contrast mode is activated, the hover state must display the focus border.

**Comment by B3nz01d** [comment]
@Tayebsed93 the reason you did not have the info is that I thought I had created this task, but created it in the wrong repo... apologies

**Comment by Tayebsed93** [comment]
@B3nz01d Thank you for this information. Does it apply only in light mode or also in dark mode? I'm asking because, according to @paulinea, it only concerns light mode (cf. https://github.com/Orange-OpenSource/ouds-flutter/issues/47).

**Comment by Tayebsed93** [comment]
This request was handled in the issue related to the Radio Button component (issue #48). Since this component was under development, the high contrast adaptation was directly integrated into that task.
As the code for the Radio Button, Checkbox, and Switch components was refactored together, the changes were automatically applied to all three.

**Comment by B3nz01d** [comment]
The android High contrast mode does not work. I've logged the following issue

- [ ] #327

**Comment by B3nz01d** [comment]
Closed during Sprint # 19

**Comment by pya35** [comment]
@B3nz01d OK for checkbox and radio button, it would be interesting to think about "Filter Chip" in selected state

**Comment by Tayebsed93** [comment]
I added the label “OK PO” because @B3nz01d previously created a bug ticket (#327) about this issue, and later marked it as resolved.
So I assume they simply forgot to update the main ticket accordingly, and that everything is actually fine.

**Comment by Tayebsed93** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (beta) (527)
Version: 0.6.0
Build type: Beta
Build details (GitHub): 155
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)

**Comment by Tayebsed93** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (527)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 155
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)
